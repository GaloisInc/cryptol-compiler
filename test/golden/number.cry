cryMain = 0xFFFFFFFFFFFFFFFFFF

//:[8]
// cryMain = 1
//cryMain = 1 : [_]
