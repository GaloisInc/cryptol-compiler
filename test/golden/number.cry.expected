4722366482869645213695

